.github/workflows/test-example-apps.yml

```diff
@@ -0,0 +1,38 @@
+name: Test Example Apps
+on:
+  pull_request:
+    branches: ['**']
```

Comment:
does it make sense to run on every pull request when the deps are coming from npm instead of local workspace? I thought this would run on release trigger